Packs/Hoxhunt/Integrations/HoxhuntV2/HoxhuntV2.yml

```diff
@@ -0,0 +1,217 @@
+      description: Returns number of removed threats.
+      type: number
+    description: Remove all threats that belong to an incident.
+    execution: true
+  - name: hoxhunt-incident-soc-feedback-send
+    arguments:
+    - name: incident_id
+      required: true
+      description: The ID of the Hoxhunt incident for which SOC feedback will be sent.
+      type: unknown
+    - name: custom_message
+      required: true
+      description: A custom message to include with the SOC feedback.
+      type: unknown
+    - name: threat_feedback_reported_at_limit
+      required: true
+      description: Datetime limit as a iso string (e.g. "2024-10-30T08:37:42.359Z").
+      type: unknown
+    outputs:
+    - contextPath: Hoxhunt.sendIncidentSocFeedback
+      description: The reporter will be informed the incident has been resolved and that no further actions are required from them.
+      type: string
+    description: Send feedback to reporters of incident about whether the reported email was safe, spam or malicious.
+  - name: hoxhunt-incident-set-sensitive
+    arguments:
+    - name: incident_id
+      required: true
+      description: The ID of the Hoxhunt incident to be marked as sensitive or not sensitive.
+      type: unknown
+    - name: is_sensitive
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - "TRUE"
+      - "FALSE"
+      description: True or False Boolean for whether the incident contains sensitive information.
+      type: unknown
+    outputs:
+    - contextPath: Hoxhunt.setIncidentSensitive
+      description: Incident sensitivity information.
+      type: string
+    description: Set incident to contain sensitive information.
+  - name: hoxhunt-incident-set-soc-classification
+    arguments:
+    - name: incident_id
+      required: true
+      description: The ID of the Hoxhunt incident to classify.
+    - name: classification
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - MALICIOUS
+      - SPAM
+      - SAFE
+      description: The SOC classification to apply to the incident.
+    outputs:
+    - contextPath: Hoxhunt.setIncidentSocClassification
+      description: Incident SOC classification information.
+      type: string
+    description: Set soc classification for an incident.
+  - name: hoxhunt-incident-update-state
+    arguments:
+    - name: incident_id
+      required: true
+      description: The ID of the Hoxhunt incident to update.
+    - name: state
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - OPEN
+      - RESOLVED
+      description: The new state of the incident.
+    outputs:
+    - contextPath: Hoxhunt.updateIncidentState
+      description: Incident state information.
+      type: string
+    description: Updates Incident state.
+  - description: Get mapping fields from remote incident. Please note that this method will not update the current incident. It's here for debugging purposes.
+    name: get-mapping-fields
```

Comment:
I understood that it's optional to add those based on the [docs](https://xsoar.pan.dev/docs/integrations/mirroring_integration#debugging). I didn't find the need to include them. We can add them if necessary, otherwise I'd just skip having them exposed.

Reply:
Ok, I was just wondering because the `get-mapping-fields` is there.